phpdotnet/phd/Package/PHP/Web.php

```diff
@@ -116,13 +117,22 @@ public function header($id) {
             if (!isset($written_toc[$filename])) {
                 $written_toc[$filename] = true;
 
-                $toc = array();
+                $toc = $toc_deprecated = array();
 
                 foreach($siblings as $sid) {
-                    $toc[] = array(
-                        Format::getFilename($sid).$ext,
-                        Format::getShortDescription($sid),
-                    );
+                    $sibling_short_desc = Format::getShortDescription($sid);
+
+                    if (!isset($this->deprecated[$sibling_short_desc])) {
```

Comment:
I prefer if (truth) over if (not true) expressions.

Please make it if(isset()) {.... } else {} :)